# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.h
# Commit: 7d270e96dfbf
# Full Hash: 7d270e96dfbfcc5992f9a1704e7d2229d1a645f3
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Adapt VideoConduit to the new libwebrtc version. r=ng,bwc
#   
#   This hooks up VideoReceiveStream::Config with a decoder factory and
#   VideoSendStream::Config with both a bitrate allocator factory and an encoder
#   factory.
# ==============================================================================

diff -r 2f1e8f5b2fc2 -r 7d270e96dfbf dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.h
--- a/dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.h	Mon Feb 08 17:23:44 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/WebrtcGmpVideoCodec.h	Mon Feb 08 18:15:07 2021 +0100
@@ -347,7 +347,7 @@
   }
 
  private:
-  RefPtr<RefCountedWebrtcVideoEncoder> mEncoderImpl;
+  const RefPtr<RefCountedWebrtcVideoEncoder> mEncoderImpl;
 };
 
 class WebrtcGmpVideoDecoder : public GMPVideoDecoderCallbackProxy {
@@ -477,7 +477,7 @@
   int32_t Release() override { return mDecoderImpl->ReleaseGmp(); }
 
  private:
-  RefPtr<WebrtcGmpVideoDecoder> mDecoderImpl;
+  const RefPtr<WebrtcGmpVideoDecoder> mDecoderImpl;
 };
 
 }  // namespace mozilla